claim/index.html

```diff
@@ -97,15 +97,15 @@ <h1>Claim HNS</h1>
   <p>This page explains how <b>github developers with over 15 followers on February 2019</b>, or in the <b>PGP WoT Strong Set</b> can claim HNS. Being able to claim does NOT imply that one is a "top open source developer", this system was optimized for a list of previously scrapeable keys (and could not be modified after the Handshake network launches without a hard fork).</p>
   <h2>System Setup</h2>
   <p>Please read through these instructions carefully, as using cryptographic blockchains are a bit unusual.</p>
-  <p>Make sure you have nodejs and npm installed first. On MacOS, please install <a href="https://brew.sh">homebrew</a> and run "<b>brew install node</b>". On debian/ubuntu, you can run "<b>sudo apt-get install nodejs npm build-essential</b>". If you run other distributions or OSes, you can probably figure this part out.</p>
+  <p>Make sure you have nodejs and npm installed first. On MacOS, please install <a href="https://brew.sh">homebrew</a> and run "<b>brew install node</b>". On debian/ubuntu, you can run "<b>sudo apt-get install nodejs npm build-essential libunbound-dev</b>". If you run other distributions or OSes, you can probably figure this part out.</p>
   <p>Next, install node-gyp: <b>npm install node-gyp</b></p>
   <h2>Download</h2>
   <p>Download <b>hsd</b>, <b>hs-client</b>, and <b>hs-airdrop</b> from <a href="https://handshake.org/download/">https://handshake.org/download/</a>. If downloaded from github, the directory structure is slightly different (hsd-2.4.0/hsd should be replaced with just hsd in these instructions).</p>
   <p>Extract hsd, hs-client, and hs-airdrop: <b>tar xvf hs*</b></p>
   <p>You may also verify the asc file if desired.</p>
   <h2>Install</h2>
-  <p>In one window, change into the hsd directory and install <b>cd hsd-2.4.0/hsd</b> and then run <b>npm install --production</b></p>.
-  <p>In the second window, change into the hs-client directory and install <b>cd hs-client-0.0.9/hs-client</b> and then run <b>npm install --production</b></p>
+  <p>In one window, change into the hsd directory and install <b>cd hsd-*/hsd</b> and then run <b>npm install --production</b></p>.
+  <p>In a second window, change into the hs-client directory and install <b>cd hs-client-*/hs-client</b> and then run <b>npm install --production</b></p>
```

Comment:
I think both of these lines are worded akwardly -- can we remove the phrase "and install" from each? It makes more sense, right? First change into the directory, then install with `npm install`...